### PR TITLE
Fix: Clear assigned doctor

### LINKED ISSUE #3227: Telemedicine Malfunction
image

- [ ] iam unable to clear the existing  assigned doctor and keep it blank

### PR DESCRIPTION
**Pull request by GokulramGHV**
Fixes #3227 

- [x]  Fixed: unable to clear the existing assigned doctor and keep it blank

### CODE REVIEW

File: src/Components/Facility/ConsultationForm.tsx [hunk 1]
```diff
@@ -552,8 +552,7 @@ export const ConsultationForm = (props: any) => {
   };
 
   const handleDoctorSelect = (doctor: UserModel | null) => {
-    doctor &&
-      doctor.id &&
+    if (doctor && doctor.id) {
```
Comment (developedBySJ) [inline src/Components/Facility/ConsultationForm.tsx]: Use optional chaining
```suggestion
    if (doctor?.id) {
```

**Review by developedBySJ — CHANGES_REQUESTED** [review]

**Comment by nihal467** [comment]
@GokulramGHV once work is completed, change the label

**Comment by GokulramGHV** [comment]
@nihal467 The work is completed

**Comment by nihal467** [comment]
@GokulramGHV when we click clear, the droplist appears, can we avoid that!

**Comment by GokulramGHV** [comment]
@nihal467 I've made the changes..

**Comment by nihal467** [comment]
image

@GokulramGHV make it responsive

**Comment by nihal467** [comment]
@GokulramGHV 
image

the clear button is covered in responsive, any better place

**Comment by GokulramGHV** [comment]
@nihal467 is this okay?
Screenshot from 2022-08-12 12-24-19

**Comment by nihal467** [comment]
@GokulramGHV can you put a cross button and try it out, rather than keeping a button saying clear?

**Comment by GokulramGHV** [comment]
@nihal467 Is this ok?
image

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@developedBySJ can you please rereview this one?

**Comment by nihal467** [comment]
@GokulramGHV merge the latest development branch and check it passes the cypress test or not

**Pull request merged**